CONTRIBUTING.md

```diff
@@ -30,9 +30,9 @@ HedgeDoc is a volunteer effort. We encourage you to pitch in and to help us maki
 
 Thanks! :heart: :heart: :heart:
 
-[issues]: https://github.com/codimd/react-client/issues
-[new_issue]: https://github.com/codimd/react-client/issues/new/choose
+[issues]: https://github.com/hedgedoc/react-client/issues
+[new_issue]: https://github.com/hedgedoc/react-client/issues/new/choose
```

Comment:
Do we want to change this already? The new repo URL doesn't exist yet and as it's our contributing information, it should probably be correct.

Reply:
Good point. I changed that back.